### ISSUE TITLE
How to start this project

### ISSUE THREAD

**Issue by xiaohui96** [edprince/remote-control#1]
Hi, i have a question about how to run this project, would you please help me? Or just run npm start?

**Comment by edprince** [comment]
Hey - looks like ncurses npm package is no longer maintained - and I can't get a working version installed successfully. If you manage that, it should just be a case of navigating into `/server` and run `node app.js` if I remember correctly. There is no `npm start` script actually written.

**Comment by xiaohui96** [comment]
Yeah, ncurses cannot be installed through npm and I am working on how to fix it...发自我的华为手机-------- 原始邮件 --------发件人： Ed Prince ***@***.***>日期： 2021年3月25日周四 中午11:59收件人： edprince/remote-control ***@***.***>抄送： xiaohui ***@***.***>, Author ***@***.***>主    题： Re: [edprince/remote-control] How to start this project (#1)
Hey - looks like ncurses npm package is no longer maintained - and I can't get a working version installed successfully. If you manage that, it should just be a case of navigating into /server and run node app.js if I remember correctly. There is no npm start script actually written.
